internal/config.go

```diff
@@ -267,16 +297,44 @@ func InitConfig() {
 		tracelog.WarningLogger.Println("We found that some variables in your config file detected as 'Unknown'. \n  " +
 			"If this is not right, please create issue https://github.com/wal-g/wal-g/issues/new")
 	}
+}
 
-	// Set compiled config to ENV.
-	// Applicable for Swift/Postgres/etc libs that waiting config paramenters only from ENV.
-	for k, v := range viper.AllSettings() {
-		val, ok := v.(string)
-		if ok {
-			if err := os.Setenv(strings.ToUpper(k), val); err != nil {
-				tracelog.ErrorLogger.Println("failed to bind config to env variable", err.Error())
-				os.Exit(1)
-			}
+func bindToEnv(k string, val string) {
+	if err := os.Setenv(strings.ToUpper(k), val); err != nil {
+		tracelog.ErrorLogger.Println("failed to bind config to env variable", err.Error())
+		os.Exit(1)
```

Comment:
And you can do it with one line.
tracelog.ErrorLogger.FatalOnError()
It err == nil than nothing happened, but if there is an error it will print logline and exit.